docs/parkiNarodowe.js

```diff
@@ -0,0 +1,177 @@
+
+    const lvlBar = document.querySelector("#myBar");
+    const lvlText = document.querySelector("#overlayText");
+
+    function fibonachiNumber(point) {
+        if (point == 0) {
+            return 0;
+        }
+        else if (point == 2 || point == 1) {
+            return 1;
+        }
+        else {
+            let loop = 1;
+            let a = 1;
+            let b = 1;
+            while (loop < point) {
+                if (loop % 2 != 0) {
+                    a = a + b;
+                }
+                else {
+                    b = a + b;
+                }
+                loop += 1;
+            }
+            if (a > b) {
+                return a;
+            }
+            return b;
+        }
+    }
+
+    function moveBar(curAmmount, maxAmmount) {
+        var width = Math.ceil(curAmmount / maxAmmount * 100);
+        lvlBar.style.width = width + "%";
+    }
+
+    moveBar(experience, expCap);
+    lvlText.innerText = experience + "/" + expCap + " XP, level " + lvl;
+    function expUp() {
+        experience += 1;
+        updateXP();
+    }
+    function updateXP() {
+        if (experience >= expCap) {
+            lvl += 1;
+            curFibonachi += 1;
+            expCap = fibonachiNumber(curFibonachi);
+            experience = 0;
+            if (expCap <= 0) {
+                console.log("Wow. Int overload... that was so unexpected... If your testing this library, then I understand how you got here, else... go touch some grass, pls");
+            }
+        }
+
+        saveXPAndLvl(experience, lvl);
+
+        moveBar(experience, expCap);
+        lvlText.innerText = experience + "/" + expCap + " XP, level " + lvl;
+    }
+
+    // Get the HTML Canvas
+    const canvas = document.querySelector("#image_canvas");
+    // Get the canvas context
+    const ctx = canvas.getContext('2d');
+
+    const name = document.querySelector('#name');
+    const points = document.querySelector('#points');
+
+    const chosen = document.querySelector('#chosen');
+
+    // Set this variable to any image source
+    const IMAGE_SRC = 'parkiNarodoweRaw.png';
+
+    // Create a new image and add it to your canvas
+    const img = new Image();
+    img.height = 700;
+    img.width = 800;
+    // Wait for the image to load before drawing to the canvas
+    img.addEventListener('load', function () {
+        ctx.drawImage(img, 0, 0);
+    }, false);
+
+    img.crossOrigin = '';
+    img.src = IMAGE_SRC;
+
+    // This is important to avoid cross origin errors
+
+    let pointCounter = 0;
+
+    let randomRegion = regions[RandomInt(regions.length)];
+
+    name.innerText = randomRegion.name;
+    points.innerText = pointCounter;
+
+    canvas.onmousedown = function (e) {
+        let mouseX = e.offsetX;
+        let mouseY = e.offsetY;
+        let colorData = ctx.getImageData(mouseX, mouseY, 1, 1).data;
+        let rgb = `${colorData[0]};${colorData[1]};${colorData[2]}`;
+        console.log(rgb);
```

Comment:
Get rid of debugging console log

Reply:
I was using it to get rgb of the thing I press, because It was easier to get that from console, than to write it while  creating map